packages/synapse-react-client/src/utils/functions/SanitizeHtmlUtils.test.ts

```diff
@@ -9,31 +8,31 @@ function createHTML(innerHTML: string) {
 
 describe('HTML Santization', () => {
   test('Prevents XSS through onload', () => {
-    const sanitized = xss("<span onload='alert('XSS')'>foo</span>", xssOptions)
+    const sanitized = sanitize("<span onload='alert('XSS')'>foo</span>")
```

Comment:
Consider running these tests against both libraries while we still use xss in SWC.

Reply:
Good idea